### PR TITLE
Motorcycles & Madness

### PR DESCRIPTION
**Pull request by ProbablyCarl**
- - -
This PR was initially intended to add keycard energy fields to the game, as might be hinted by the branch name. I've discarded that for now, however, and hadn't bothered to change the branch name.
- - -
Balance:
 - Initial introduction of Motorcycles as a faction based object. The NCR gets a Ranger bike, with the key in the Veteran's quarters. The Legion must brave the Renegade camp to get their bike. Otherwise, a rusted bike, with ten percent slowdown, is present in the fire station. That's three bikes in total, one per map. This is likely to be adjusted, as the values for the bikes aren't tweaked and you can snake with them.
 - HMG Mutant given rebalances, post viewing of how it functions in game.
 - Rock Springs firewatch tower given a proper perimeter wall, to make utilizing it as a player base far easier.
- - -
QoL:
 - Legion ORM moved to provide access.
 - Legion given a mining equipment vendor for use with the ORM.
 - Camera console in prison removed.
 - Rooftop added to the shack in Rock Springs, south of the bee farm.
- - -

### CODE REVIEW

File: code/modules/vehicles/_vehicle.dm [hunk 1]
```diff
@@ -218,17 +220,18 @@
...
+*/
 	if(!inserted_key)
 		usr.visible_message("<span class = 'notice'>There is no key.</span>")
 		return
 
-	src.verbs += /obj/vehicle/proc/StopEngine
-	src.verbs -= /obj/vehicle/proc/StartEngine
-
 	if(usr)
 		usr.visible_message("[usr] start engine of [src].", "You start engine.")
```
Comment (out-of-phaze) [inline code/modules/vehicles/_vehicle.dm:236]:
```suggestion
		M.visible_message("[M] starts the engine of [src].", "You start the engine.")
```

File: code/modules/vehicles/_vehicle.dm [hunk 2]
```diff
@@ -218,17 +220,18 @@
...
-
+*/
 	if(!inserted_key)
 		usr.visible_message("<span class = 'notice'>There is no key.</span>")
 		return
 
-	src.verbs += /obj/vehicle/proc/StopEngine
-	src.verbs -= /obj/vehicle/proc/StartEngine
-
 	if(usr)
```
Comment (out-of-phaze) [inline code/modules/vehicles/_vehicle.dm:235]:
```suggestion
	if(M)
```

File: code/modules/vehicles/_vehicle.dm [hunk 3]
```diff
@@ -218,17 +220,18 @@
...
+	src.verbs += /obj/vehicle/proc/StopEngine
+	src.verbs -= /obj/vehicle/proc/StartEngine
+/*
 	if(!M.buckled)
 		usr.visible_message("<span class = 'notice'>Sit on [src] to do this.</span>")
 		return
-
+*/
 	if(!inserted_key)
 		usr.visible_message("<span class = 'notice'>There is no key.</span>")
```
Comment (out-of-phaze) [inline code/modules/vehicles/_vehicle.dm:232]:
```suggestion
		M.visible_message(span_notice("There is no key."))
```

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
Some issues with the motorcycle code.

**Comment by ProbablyCarl** [comment]
Verb movement was unintentional. Good catch.

**Review by out-of-phaze — APPROVED** [review]
Looks good. TODO: Balance and code quality pass on motorcycles.

**Pull request merged**